app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+import os
+
+
+def _check_path(new_path: str) -> None:
+    cur_path = ""
+    for directory in new_path.split("/"):
+        cur_path = os.path.join(cur_path, directory)
+        if not os.path.exists(cur_path):
+            os.mkdir(cur_path)
+
+
+def move_file(command: str) -> None:
+    try:
+        operation, old_file_loc, new_file_loc = command.split()
+        if not os.path.isfile(old_file_loc):
+            raise ValueError
+    except ValueError:
+        print(
+            "Please check the input, "
+            "it should be like 'mv file some_dir/new_file'. "
+            "The file to move cannot be directory!"
+        )
+        return
```

Comment:
For what do you raise and then handle the error?)

Reply:
this was check in case when split() returns more or less variables than we need. Also i wanted to raise the same error in case when name of directory was passed as a file to move. nvm i removed it